addons/binding/org.openhab.binding.meteostick/README.md

```diff
@@ -139,6 +139,10 @@ then
 	val pw = 'your_password'
 	val sdf = new SimpleDateFormat('yyyy-MM-dd HH:mm:ss')
 	sdf.setTimeZone(TimeZone.getTimeZone('UTC'))
+	val double rh = DavisVantageVueHumidity.getStateAs(DecimalType).doubleValue
+	val double tempc = DavisVantageVueOutdoorTemperature.getStateAs(QuantityType).toUnit('°C').doubleValue
+	val double dewptc = 243.04 * (Math.log(rh/100) + ((17.625 * tempc) / (243.04 + tempc))) / (17.625 - Math.log(rh/100) - ((17.625 * tempc) / (243.04 + tempc)))
+	val double dewptf = new QuantityType(dewptc, CELSIUS).toUnit('°F').doubleValue
```

Comment:
Did you also check if additional imports need to be added for the new code?

The link http://wiki.wunderground.com/index.php/PWS_-_Upload_Protocol no longer gives details about the upload protocol. Maybe a good alternative is https://feedback.weather.com/customer/en/portal/articles/2924682-pws-upload-protocol?b_id=17298 ?

Reply:
Thanks for catching that stale doc link, a pretty recent change!  No new imports needed -- CELSIUS etc. are now auto-imported into rules and scripts.